spec/features/submissions_spec.rb

```diff
@@ -48,10 +51,10 @@
         before do
           Submission.create!(person_id: Person.first, status: "odrzucony", polish_title: "Alicja w krainie czarów", english_title: "Alice 
             in Wonderland", polish_abstract: "Słów parę o tej bajce", english_abstract: "Little about that story", polish_keywords: "alicja",
-            received: "19-01-2016", language: "polski")
+            received: "19-01-2016", language: "polski", issue_id: Issue.first)
           Submission.create!(person_id: Person.first, status: "do poprawy", polish_title: "W pustyni i w puszczy", english_title: "Desert 
             and something", polish_abstract: "Porywająca lektura", english_abstract: "Super lecture", polish_keywords: "pustynia",
-            received: "19-01-2016", language: "polski")	
+            received: "19-01-2016", language: "polski", issue_id: Issue.first)	
```

Comment:
Poza tym - jeśli chcemy, żeby "W pustyni i w puszczy" się nie wyświetliło to musimy przypisać inny numer!